docs/source/technical_tutorials/accessing.rst

```diff
@@ -9,3 +9,4 @@ Access
   access/accessing_external_data.ipynb
   access/accessing_cod.ipynb
   access/edav_wcs_data.ipynb
+  access/accessing_LVIS_data.ipynb
```

Comment:
Should this go under science tutorials?

Reply:
My first thought is under technical tutorials, but we also don't have any LVIS under science, so I guess it could go either way? If we do science, we should probably rename it to something like "LVIS Access and Explore" just to fit under the naming format better

Reply:
Updating it to be a Science notebook under the name - `LVIS Access and Explore`. Thanks!